portal/src/graphql/adminapi/UserDetailsAccountSecurity.module.css

```diff
@@ -1,4 +1,6 @@
-$descriptionTextColor: #888888;
+:root {
+  --descriptionTextColor: #888888;
```

Comment:
Again, `:root` should not be used in CSS module. Move the custom variable declaration to `.root`.